app/services/data/get-app-workloads.js

```diff
@@ -73,7 +78,11 @@ module.exports = function (initialId, maxId, batchSize, workloadReportId) {
                           row.tiers_total_cases,
                           row.warrants_total,
                           row.unpaid_work_total,
-                          row.overdue_terminations_total)
+                          row.overdue_terminations_total,
+                          row.t2a_tiers_total_cases,
+                          row.t2a_warrants_total,
+                          row.t2a_unpaid_work_total,
+                          row.t2a_overdue_terminations_total)
```

Comment:
Also need to be included in `select`?

Reply:
Have these been added?

Reply:
Added the other fields and not the weighting. Added that in now.